script/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e
+
+# Build LightTable app and CLI and place in builds/.
+# Specify $VERSION to override default build version.
+# This script primarily installs dependencies and sets up
+# the app before calling build-app.sh to build it.
+
+# Check if lein is installed
+lein version >/dev/null 2>&1 || { echo >&2 "Please install leiningen before running this script."; exit 1; }
+if [ "$(echo `lein version` | grep 'Leiningen \(1.\|2.0\)')" ]; then
+  echo "lein version must be 2.1 or above. Do a lein upgrade first"; exit 1;
+fi
+
+# Check if npm is installed
+npm --version >/dev/null 2>&1 || { echo >&2 "Please install npm before running this script."; exit 1; }
+
+# Ensure we start in project root
+cd "$(dirname "${BASH_SOURCE[0]}")"; cd ..
+
+# Ensure we have current version of atom-shell
+pushd shell
+  npm install -g grunt-cli
```

Comment:
I agree with Kolya on this actually, I'd just recommend not installing it globally.